zipkin4net-owin/Criteo.Profiling.Tracing.Middleware/Criteo.Profiling.Tracing.Middleware.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{6A57C709-4443-4735-94CF-19DE60B00F59}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Criteo.Profiling.Tracing.Middleware</RootNamespace>
```

Comment:
I would vote in favor of Criteo.Profiling.Tracing.Owin.Middleware.  Asp .Net is already published under the name https://www.nuget.org/packages/Criteo.Profiling.Tracing.Middleware/ .
This is going to create very long directory names. To clarify things a bit we could revamp (In another story) the directory structure. For example we shouldn't have to repeat zipkin4net in front of every root directory. Same goes for Criteo.Profiling.Tracing. With a good structure, package naming should become simpler and more coherent.

What do you want to do about the existing Criteo.Profiling.Tracing.Middleware ?